libs/filabridge/include/private/filament/UibStructs.h

```diff
@@ -149,8 +149,7 @@ struct alignas(256) PerRenderableUib { // NOLINT(cppcoreguidelines-pro-type-memb
     static constexpr utils::StaticString _name{ "ObjectUniforms" };
     math::mat4f worldFromModelMatrix;
     math::mat3f worldFromModelNormalMatrix;   // this gets expanded to 48 bytes during the copy to the UBO
-    alignas(16) math::float4 morphWeights;    // morph weights (we could easily have 8 using half)
```

Comment:
store the count here  (instead of morphWeights) and instead of storing it in `PerRenderableMorphingUib`

Reply:
Thanks for guide. It's done.